src/views/campaigns/userCampaigns.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="user-campaigns" v-for="campaign in userCampaigns" v-bind:key="campaign.id">
+    <section class="campaign-card">
+      <section>
+      <div class="img-div"><img :src=campaign.image /></div>
+      <div class="campaign-info">
```

Comment:
This div looks different, is it missing a closing tag?

Reply:
Ohh good catch.

Reply:
Whoops, yea did the templating on the fly.

Reply:
I did the templating on the fly so some of the indenting is wrong, but the closing tags are there. I don't think it would build otherwise.